app/views/components/unordered_list.scala.html

```diff
@@ -16,10 +16,10 @@
 
 @this()
 
-@(msg: String*)(implicit messages: Messages)
+@(items: Html*)(implicit messages: Messages)
 
 <ul class="govuk-list govuk-list--bullet govuk-!-margin-bottom-6">
-    @msg.map { listItem =>
-        <li>@messages(listItem)</li>
+    @items.map { item =>
```

Comment:
is this gonna have unintended consequences on any other page?

Reply:
Hey @AdamCoops the `UL` component is not used elsewhere

Reply:
👍